cmd/crd-gen-tool/main.go

```diff
@@ -71,6 +71,10 @@ func main() {
 	clusterApiVersion := env("CLUSTER_API_PROVIDER_VERSION", "v1.2.5")
 	awsProviderVersion := env("AWS_PROVIDER_VERSION", "v2.0.0")
 
+	if err := os.MkdirAll(crdOutputDir, 0755); err != nil {
+		log.Fatalf("MkdirAll %s got error %+v", crdOutputDir, err)
```

Comment:
should fail fast return?

Reply:
Yes, If the folder is not created successfully, writing the file will fail.